container-core/src/main/resources/configdefinitions/vip-status.def

```diff
@@ -10,3 +10,6 @@ statusfile string default="share/qrsdocs/status.html"
 
 ## Not used TODO: Remove on Vespa 8
 initiallyInRotation bool default=true
+
+## Require all clusters UP to set container UP
+requireAllUp bool default=false
```

Comment:
Is this a public config ? If so we should be very careful about it. Remember that we are not able to give any warnings or deprecations or anything to users ahead of time.

I would guess that we need a a list of mandatory clusters and a matchtype
require {ANY, ALL}

I think that this requires an architect meeting.